lib/backend.js

```diff
@@ -225,6 +226,18 @@ exports.createServer = (dynamodb, docClient) => {
       .catch(next)
   })
 
+  app.delete('/tables/:TableName/all', (req, res) => {
+    purgeTable(req.params.TableName, dynamodb)
+      .then(() => {
+        res.status(200).end()
+      })
+      .catch((error) => {
+        console.log(`An error has occurred wile trying to purge table ${req.params.TableName}`)
+        res.status(500).end()
```

Comment:
I think it's better to do like other endpoints do and just pass error to `next`. Will give a stack trace and more detailed error.

Reply:
Actually default handler doesn't seem to post any error. It just returns with 500 error.
So Maybe we should define error handler after all the endpoints so that it posts the real error to the console.